invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/accordion.overrides

```diff
@@ -26,6 +26,11 @@ div.affiliations.accordion div.title button.down {
 
 
 .ui.accordion {
+
+  &.panel#preview {
+    overflow: hidden;
```

Comment:
wouldn't this cause some files to be "cut" in the preview ? isn't it better to use `object-fit: contain`?